waku/v2/protocol/peer_exchange/protocol.go

```diff
@@ -100,6 +100,9 @@ func (wakuPX *WakuPeerExchange) onRequest() func(network.Stream) {
 			wakuPX.metrics.RecordError(rateLimitFailure)
 			wakuPX.log.Error("exceeds the rate limit")
 			// TODO: peer exchange protocol should contain an err field
+			if err := stream.Reset(); err != nil {
+				wakuPX.log.Error("resetting connection", zap.Error(err))
```

Comment:
Let's not log an error here, because this is not an error condition on peerExchange service node side, maybe we can log debug.

Reply:
Hmm this is an error coming out of `stream.Reset()` - what is the difference between this and all the other handling of `stream.Reset()` errors later? (I don't mind changing, just trying to understand the reasoning)

Reply:
Looking at other instances of `stream.Reset()` in the server side, we're using `Error` there.

Reply:
my bad, this is fine. 
i wanted to indicate if we should log an error in previous line above `101` when rate limit is exceeded. Because a node may get flooded with error logs if someone is trying to constantly use it despite rate limit error. 
Minor comment, can be addressed later as well.